includes/bp-media-function.php

```diff
@@ -602,10 +602,9 @@ function bp_media_group_image_link( $id, $user_id ) {
 	echo bp_media_get_group_image_link( $id, $user_id );
 }
 
-	function bp_media_get_group_image_link( $id, $user_id ) {
-		return  bp_core_get_user_domain( $user_id ) . BP_MEDIA_SLUG . '/image/' . $id;
-	}
-
+function bp_media_get_group_image_link( $id, $user_id ) {
+	return  bp_core_get_user_domain( $user_id ) . BP_MEDIA_SLUG . '/image/' . $id;
```

Comment:
Also a FYI regarding the indentation of some of these function declarations. It is a holdover of sorts from how BuddyPress core groups together their "echo" and "return" functions. They indent the declaration for the return versions.